tpgtools/overrides/compute/samples/firewallpolicyrule/meta.yaml

```diff
@@ -0,0 +1,19 @@
+# meta.yaml
```

Comment:
Hi @shuyama1, thanks for the review. This file `mmv1/third_party/terraform/website/docs/r/compute_firewall_policy_rule.html.markdown` I thought it was automatically generated by the tpgtools, because there is a header mentioning that it's "Auto Generated Code".

Reply:
@shuyama1 this doc was updated few days ago with the PR mentioned, no this PR just add tests for the scenarios and there is updates in small parts of the doc.